src/lib/custom-resources/cdk-guardduty-admin-setup/runtime/package.json

```diff
@@ -5,15 +5,15 @@
   "main": "dist/index.js",
   "types": "src/index.ts",
   "scripts": {
-    "build": "pnpx esbuild --minify --bundle --platform=node --target=node14 --external:aws-sdk --outfile=./dist/index.js src/index.ts",
+    "build": "pnpx esbuild --minify --bundle --platform=node --target=node14  --outfile=./dist/index.js src/index.ts",
     "lint:typecheck": "pnpx tsc --noEmit",
     "lint:eslint": "pnpx eslint '{cdk,lib,src}/**/*.{js,ts}'"
   },
   "dependencies": {
     "@aws-accelerator/custom-resource-cfn-utils": "workspace:*",
     "@aws-accelerator/custom-resource-runtime-cfn-response": "workspace:*",
     "aws-lambda": "1.0.6",
-    "aws-sdk": "2.944.0"
+    "aws-sdk": "2.1217.0"
```

Comment:
I had to bump up the sdk version as 2.944.0 does not contain the GuardDuty Kubernetes API.